Form-Controls/index.html

```diff
@@ -14,14 +14,50 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
+    <div>
+      <lable for="name">Name</lable>
+      <input type="text" name=" name" id="name" required> 
```

Comment:
This isn't so much of a problem here because we aren't giving the form an `action` attribute as part of the exercise, but do pay attention to extra spaces within HTML element attribute values (in this case `name=" name"`) as the spaces here are significant and would make a difference to how the form was, for example, submitted as a request to a web application.

Reply:
Yes, absolutely, I have deleted the extra space before name. They are so tricky. ;)